packages/components/src/components/hds/app-side-nav/list/link.hbs

```diff
@@ -0,0 +1,52 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::AppSideNav::List::Item>
+  <Hds::Interactive
+    class="hds-app-side-nav__list-item-link {{if @isActive 'active'}}"
+    tabindex={{if @isActive "-1"}}
```

Comment:
@shleewhite @alex-ju I added a negative tab index to prevent tabbing to the "active" Link from Jory's feedback above. Is this an acceptable solution?

Reply:
Hmm.. I think we should discuss this. It would be an a11y violation to remove a non-disabled interactive element from the tab order. What are the concerns with having it focusable? Just that a ring is visible?

Reply:
Perhaps it would be better to try to change it from a link/button to plain text when it's "active"? I don't know the reasoning behind why it was kept as a link in the SideNav when it links to the same page and is thus not useful (or I assume it's not useful).

@alex-ju do you have any thoughts around this?

Reply:
I discussed with @jorytindall and I'm going to undo this. This means that the "active" link will have a focus border again as it did in the SideNav.

We can discuss if we want to change this behavior as a follow on. (To avoid over-complicating this PR.)

Reply:
Thanks for the update, Kristin. If we don't want it to be focused I see two options: either render it as a disabled button (meaning `aria-current="page"` will not be surfaced), or it could be plain text (or even better the h2 that would be overarching the rest of the SideNav titles, as Lee suggested in another comment above).